src/com/lftechnology/java/training/sanish/generics/constants/PersonList.java

```diff
@@ -0,0 +1,32 @@
+package com.lftechnology.java.training.sanish.generics.constants;
+
+import java.io.Serializable;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.Serializable'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)